src/LinuxTracing/UprobesUnwindingVisitor.cpp

```diff
@@ -528,22 +528,50 @@ void UprobesUnwindingVisitor::Visit(uint64_t event_timestamp,
 
 void UprobesUnwindingVisitor::Visit(uint64_t event_timestamp,
                                     const UserSpaceFunctionEntryPerfEventData& event_data) {
-  function_call_manager_->ProcessFunctionEntry(event_data.tid, event_data.function_id,
-                                               event_timestamp, std::nullopt);
+  const auto& tid_to_root_namespace_tid_it = tid_to_root_namespace_tid_.find(event_data.tid);
+  if (tid_to_root_namespace_tid_it == tid_to_root_namespace_tid_.end()) {
+    ORBIT_ERROR_ONCE(
```

Comment:
Of course this is logged (at most) once per execution of `OrbitService`, not per capture. It's totally fine.

Reply:
ack